improver_tests/wxcode/wxcode/test_ModalFromGroupings.py

```diff
@@ -325,11 +329,10 @@ def test_expected_values_ignore_intensity(
         intensity_categories = {}
         for key in INTENSITY_CATEGORIES.keys():
             intensity_categories[key] = [i for i in reversed(INTENSITY_CATEGORIES[key])]
+    if not ignore_intensity:
+        intensity_categories = None
```

Comment:
I think this could be re-factored along the lines of:

`if ignore_intensity: intensity_categories = INTENSITY_CATEGORIES.copy()`
`else: intensity_categories = None`

with the `reverse_intensity_dict` block of code included under `if TRUE`

Reply:
Amended as suggested.